lib/article_json/export/apple_news/exporter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(elements)
         # Return the components section of an Apple News Article as JSON
         # @return [String]
         def to_json
-          { components: components }.to_json
+          { components: components.flatten }.to_json
```

Comment:
Maybe a comment why we're calling `flatten` here would be helpful?

Reply:
I considered that but thought I might be over-explaining. I will happily add it now.

Reply:
Added a comment to explain `.flatten` and correct the YARD `@return` for the `components` method to reflect that it returns an array rather than a string.